README.md

```diff
@@ -30,10 +30,14 @@ milestones:                             # If there are no issues do not include
       cannot use the service reliably. This does not meet WCAG success criterion
       X.Y.Z (Criterion Description).
     date: 2020-09-30                    # The date that this issue will be fixed by
-serviceLastTestedDate: 2019-09-15       # In ISO format YYYY-MM-DD
+serviceLastTestedDate: 2019-09-15       # In ISO format YYYY-MM-DD. If you statement has not yet been tested, use 2020-11-30
```

Comment:
Could we make this optional rather than having users adding the arbitrary date?

Reply:
Interesting question, I did have a think about this one. This field is only optional for non-compliant services, otherwise it is required. We would have to think of a fallback behaviour if this isn't provided for fully or partially compliant services, or start adding the business logic into our YAML parser that this property is mandatory if the statement is fully or partially compliant.

Thoughts? We have so far managed to avoid adding business logic to the parser.

Reply:
I think make it optional and we can enforce the business rule via reporting.

Reply:
I'm really wary about encouraging users to enter data that is fundamentally incorrect.

Reply:
`serviceLastTestedDate` is now an optional parameter. If it's not defined for a fully or partially compliant statement, the message about when the statement was last tested is not displayed.